website/content/docs/getting-started.md

```diff
@@ -178,6 +178,8 @@ You can open Heron UI in your browser at [http://localhost:8889](http://localhos
 and see something like this upon successful submission of the topology:
 ![alt tag](/img/heron-ui.png)
 
+Note: ``heron-tracker`` (in Step 4) should be running at the same time so that Heron UI can work properly. 
```

Comment:
Since Step 3 is to start Heron Tracker before Step 4 to start the UI, I don't think this comment is needed. It's assumed that the reader is not skipping steps.